ui/src/treeDataProvider/AzureTreeDataProvider.ts

```diff
@@ -79,13 +79,13 @@ export class AzureTreeDataProvider implements TreeDataProvider<IAzureNode>, Disp
         return this._onNodeCreateEmitter.event;
     }
 
-    public getTreeItem(node: IAzureNode): TreeItem {
+    public getTreeItem(node: AzureNode): TreeItem {
```

Comment:
what's this change for?

Reply:
I don't think node.iconPath or node.label are worth exposing outside the ui package, so I changed it from `IAzureNode` (external interface) to `AzureNode` (internal class)